package.json

```diff
@@ -42,6 +44,7 @@
 		"@testing-library/react": "^13.1.1",
 		"@types/jest": "^28.1.3",
 		"@types/react": "^18.0.0",
+		"axios": "^0.27.2",
```

Comment:
I think `axios` would fall under `dependencies` instead of `devDependencies` since we are actually using it in our library code. I got the idea from this stackoverflow: [https://stackoverflow.com/questions/48288032/axios-dependencies-or-devdependencies](https://stackoverflow.com/questions/48288032/axios-dependencies-or-devdependencies). 

Let me know what you think

Reply:
yeah I agree, we had it in dep before but with all the messing around, I forgot to commit it